libs/wingc/src/ast.rs

```diff
@@ -259,7 +259,7 @@ pub struct Class {
 	pub name: Symbol,
 	pub fields: Vec<ClassField>,
 	pub methods: Vec<(Symbol, FunctionDefinition)>,
-	pub constructor: Constructor,
+	pub constructor: Option<Constructor>,
```

Comment:
nit: I wonder if we should rename this to `initializer`? (Aim for consistent phrasing across our docs / codebase)